### PR TITLE
users: create stdiscosrv manpage

### PR DESCRIPTION
**Pull request by aviau**

### CODE REVIEW

File: users/stdiscosrv.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+::
+
+    stdiscosrv [-cert=<file>] [-db-backend=<string>] [-db-dsn=<string>] [-debug] [-http] [-key=<string>]
+               [-limit-avg=<int>] [-limit-burst=<int>] [-limit-cache=<int>] [-listen=<address>]
+               [-stats-file=<file>]
+
+Description
+-----------
+
+Syncthing relies on a discovery server to find peers. Anyone can run a discovery
```
Comment (aviau) [inline users/stdiscosrv.rst]: I am sure this is not the best description. You might be better than me to explain this (and not lie! which I might by accident.)

Comment (calmh) [inline users/stdiscosrv.rst]: "... to find peers over the Internet" should be good enough.

**Comment by calmh** [comment]
Currently adds no real value, I'll see if I can point to a few sections on how to actually deploy and so on when I'm on a real computer

**Comment by calmh** [comment]
@st-review merge it

**Comment by st-review** [comment]
@calmh: Merge failed:

```
$ git fetch -f origin refs/pull/203/head:pr-203
$ git fetch -f origin master:orig/master
From github.com:syncthing/docs
   a9b3fef..a1ac0aa  master     -> orig/master
$ git reset --hard
HEAD is now at a1ac0aa users: create strelaysrv manpage
$ git checkout master
Already on 'master'
Your branch is up-to-date with 'origin/master'.
$ git reset --hard orig/master
HEAD is now at a1ac0aa users: create strelaysrv manpage
$ git clean -fxd
$ git merge --squash --no-commit pr-203
Auto-merging users/stdiscosrv.rst
Auto-merging conf.py
CONFLICT (content): Merge conflict in conf.py
Squash commit -- not updating HEAD
Automatic merge failed; fix conflicts and then commit the result.

```

**Comment by calmh** [comment]
@aviau fix the merge conflict please and merge at will

**Comment by aviau** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as aae1a525f167c70c82f30501f9b8566c331d75a5. Thanks, @aviau!

**Pull request closed without merging**